### PR TITLE
dev-python/uvloop: Version bump to 0.5.3

### PR DESCRIPTION
**Pull request by elprans**
Gentoo-Bug: https://bugs.gentoo.org/593082

CC: @gentoo/proxy-maint

Package-Manager: portage-2.2.28

### CODE REVIEW

File: dev-python/uvloop/uvloop-0.5.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="doc examples test"
+
+CDEPEND=">=dev-libs/libuv-1.9.0:="
+DEPEND="
+	${CDEPEND}
```
Comment (SoapGentoo) [inline dev-python/uvloop/uvloop-0.5.3.ebuild]: ditch `CDEPEND`. Put the `CDEPEND` in `RDEPEND`, and include `RDEPEND` in `DEPEND`

File: dev-python/uvloop/uvloop-0.5.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+RDEPEND="${CDEPEND}"
+
+src_prepare() {
+	cat <<EOF >> setup.cfg
+[build_ext]
+use-system-libuv=1
+EOF
+	distutils-r1_src_prepare
+}
```
Comment (SoapGentoo) [inline dev-python/uvloop/uvloop-0.5.3.ebuild]: If you only use `distutils-r1` (which you do), then the general principle is not override `src_prepare()`, but instead `python_prepare_all`.

File: dev-python/uvloop/uvloop-0.5.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+python_test() {
+	esetup.py test
+}
+
+python_install_all() {
+	if use examples; then
+		insinto "/usr/share/doc/${PF}"
+		docompress -x "/usr/share/doc/${PF}/examples"
```
Comment (SoapGentoo) [inline dev-python/uvloop/uvloop-0.5.3.ebuild]: chuck the `insinto` and replace `doins` with `dodoc`. Same effect, more concise.

**Comment by elprans** [comment]
@SoapGentoo nits fixed.  Thanks for reviewing!

**Comment by monsieurp** [comment]
@gentoo/python

File: dev-python/uvloop/uvloop-0.5.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+python_test() {
+	esetup.py test
+}
+
+python_install_all() {
+	if use examples; then
+		dodoc -r examples
+	fi
```
Comment (SoapGentoo) [inline dev-python/uvloop/uvloop-0.5.3.ebuild]: Now that I see this, can you further compress this down to `use examples && dodoc -r examples`? Soz for the second round.

Comment (elprans) [inline dev-python/uvloop/uvloop-0.5.3.ebuild]: @SoapGentoo: done

**Pull request closed without merging**